local-setup/README.md

```diff
@@ -0,0 +1,36 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make`
+- integritee-node built with `cargo build --release`
```

Comment:
Needs to be built with `--features skip-ias-check`

Reply:
oh right, I forgot about that. Thanks. Will be adding that.

Reply:
Done, with an additional comment. Please check it out :)